crypto/kzg/src/trusted_setup.rs

```diff
@@ -21,117 +11,36 @@ struct G2Point([u8; BYTES_PER_G2_POINT]);
 /// See https://github.com/ethereum/consensus-specs/blob/dev/presets/mainnet/trusted_setups/trusted_setup_4096.json
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct TrustedSetup {
+    #[serde(rename = "g1_monomial")]
+    g1_monomial_points: Vec<String>,
     #[serde(rename = "g1_lagrange")]
-    g1_points: Vec<G1Point>,
+    g1_lagrange_points: Vec<String>,
     #[serde(rename = "g2_monomial")]
-    g2_points: Vec<G2Point>,
+    g2_monomial_points: Vec<String>,
 }
 
 impl TrustedSetup {
-    pub fn g1_points(&self) -> Vec<[u8; BYTES_PER_G1_POINT]> {
-        self.g1_points.iter().map(|p| p.0).collect()
-    }
-
-    pub fn g2_points(&self) -> Vec<[u8; BYTES_PER_G2_POINT]> {
-        self.g2_points.iter().map(|p| p.0).collect()
-    }
-
-    pub fn g1_len(&self) -> usize {
-        self.g1_points.len()
-    }
-}
-
-impl Serialize for G1Point {
-    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
-    where
-        S: serde::Serializer,
-    {
-        let point = hex::encode(self.0);
-        serializer.serialize_str(&point)
-    }
-}
-
-impl Serialize for G2Point {
-    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
-    where
-        S: serde::Serializer,
-    {
-        let point = hex::encode(self.0);
-        serializer.serialize_str(&point)
+    pub fn g1_lagrange_points(&self) -> Vec<u8> {
+        self.g1_lagrange_points.iter().flat_map(|p| {
+            let stripped = strip_prefix(p);
+            hex::decode(stripped).expect("expected g1 lagrange points to be well formed hex strings")
+        }).collect()
+    }
+    pub fn g1_monomial_points(&self) -> Vec<u8> {
+        self.g1_monomial_points.iter().flat_map(|p| {
+            let stripped = strip_prefix(p);
+            hex::decode(stripped).expect("expected g1 monomial points to be well formed hex strings")
+        }).collect()
+    }
+    pub fn g2_monomial_points(&self) -> Vec<u8> {
+        self.g2_monomial_points.iter().flat_map(|p|{
+            let stripped = strip_prefix(p);
+            hex::decode(stripped).expect("expected g2 monomial points to be well formed hex strings")
+        }).collect()
```

Comment:
Small nit, I've been trying to use consistent order for these: g1_monomial, g1_lagrange, g2_monomial.

Can we move the middle function to the top?